doc/docs/deployment/digital-ocean-apps.md

```diff
@@ -0,0 +1,185 @@
+---
+sidebar_position: 1
+---
+
+# Digital Ocean Apps
+
```

Comment:
```suggestion
There is a simplified deployment type without Kubernetes. This type is **recommended** for most new applications because it allows you to set up infrastructure faster and doesn't require additional DevOps knowledge from the development team. You can switch to a more complex Kubernetes solution when your application will be at scale.
```